presto-hive/src/main/java/com/facebook/presto/hive/HivePartitionKey.java

```diff
@@ -51,6 +55,11 @@ public String getValue()
         return value;
     }
 
+    public int getEstimatedSizeInBytes()
+    {
+        return INSTANCE_SIZE + name.length() * Character.BYTES + value.length() * Character.BYTES;
```

Comment:
This is long and hard to read, add parentheses

Reply:
I disagree on this one. I find it harder to read with parenthesis.

I will address all other comments. I noticed the build failure and fixed them myself. Apparently, I forgot to push it.